File: a_easy/24_swap_case/swap_case_custom.rb
def swap_case(string, length)
  array = string.split(//)

  for i in 0..length do
    if array[i].match(/[a-z]/)
      array[i] = array[i].upcase
    elsif array[i].match(/[A-Z]/)
      array[i] = array[i].downcase
    end
  end

  array.join
end

File: a_easy/24_swap_case/swap_case_custom.js
function swapCase(string, length) {
  let array = string.split("");

  for (i = 0; i < length; i++) {
    if (array[i].match(/[a-z]/)) {
      array[i] = array[i].toUpperCase();
    } else if (array[i].match(/[A-Z]/)) {
      array[i] = array[i].toLowerCase();
    }
  }

  return array.join("");
}
